wire-tests/src/test/java/com/squareup/wire/java/KotlinGeneratorTest.kt

```diff
@@ -7,7 +7,7 @@ import org.junit.Test
 import kotlin.test.assertTrue
 
 class KotlinGeneratorTest {
-  @Test fun basic() {
+  @Test fun `basic`() {
```

Comment:
Don't think these test name changes are needed, and they're outside of the scope of the PR